signing-jobs/build/sign-artifacts/Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = release.buildlib
+    def commonlib = release.commonlib
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
```

Comment:
I want these to be infinite retention. Not sure if this is how to do it, but we can fix that in post

Reply:
nice.